### PR TITLE
Feature/sdl 0262 new vehicle data seat occupancy

### PR DESCRIPTION
**Pull request by LitvinenkoIra**
Implements https://github.com/smartdevicelink/rpc_spec/issues/209

This PR **ready** for review.

### Summary
Add new vehicle data SeatOccupancy

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by KhrystynaDubovyk** [comment]
@atiwari9 please review this PR

File: README.md [hunk 1]
```diff
@@ -2267,6 +2268,23 @@ Describes the status of a window of a door/liftgate etc.
...
+| ---------- | ---------- |:-----------: |:-----------:|
+|`seatLocation`|SeatLocation|True||
+|`conditionActive`|Boolean|True||
+
+### SeatOccupancy
+##### Parameters
+
+| Value |  Type | Mandatory | Description |
+| ---------- | ---------- |:-----------: |:-----------:|
+|`seatsOccupied`|SeatStatus|False||Seat status array containing location and whether the seats are occupied.
```
Comment (atiwari9) [inline README.md]: Please correct the table

**Review by atiwari9 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LitvinenkoIra) [reply README.md]: @atiwari9 Done, please check the latest updates

**Review by atiwari9 — APPROVED** [review]
Approved.

**Comment by LitvinenkoIra** [comment]
@JackLivio This PR is ready for Livio review. Thank you!

File: MOBILE_API.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" standalone="no"?>
 <?xml-stylesheet type="text/xml" href="protocol2html.xsl"?>
 
-<interface name="SmartDeviceLink RAPI" version="7.0.0" minVersion="1.0" date="2020-11-04">
+<interface name="SmartDeviceLink RAPI" version="7.1.0" minVersion="1.0" date="2020-11-04">
```
Comment (jacobkeeler) [inline MOBILE_API.xml]: Merge in the latest develop branch to remove the need for this change

File: MOBILE_API.xml [hunk 3]
```diff
@@ -4685,6 +4686,21 @@
         </param>
     </struct>
 
+    <struct name="SeatStatus" since="7.1">
+      <description>Describes the status of a parameter of seat.</description>
+      <param name="seatLocation" type="SeatLocation" mandatory="true"/>
+      <param name="conditionActive" type="Boolean"  mandatory="true"/>
```
Comment (jacobkeeler) [inline MOBILE_API.xml]:
```suggestion
      <param name="conditionActive" type="Boolean" mandatory="true"/>
```

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

File: MOBILE_API.xml (hunk 3 above)
Comment (LitvinenkoIra) [reply MOBILE_API.xml]: @jacobkeeler Done in 476cb36e4f08c8392a247bcdbf6e6c1541b76a9b

File: MOBILE_API.xml (hunk 2 above)
Comment (LitvinenkoIra) [reply MOBILE_API.xml]: Done

**Review by jacobkeeler — APPROVED** [review]

**Comment by jacobkeeler** [comment]
@LitvinenkoIra please fix conflicts

**Comment by LitvinenkoIra** [comment]
@jacobkeeler Done in afc04a4

**Pull request merged**